data/processed/hesa/total_stem_students.csv

```diff
@@ -0,0 +1,201 @@
+2016,UKD3,2018,27470.0
+2017,UKD3,2018,27465.0
+2018,UKD3,2018,27505.0
+2014,UKJ3,2018,19695.0
+2015,UKJ3,2018,20180.0
+2016,UKJ3,2018,20725.0
+2017,UKJ3,2018,20445.0
+2018,UKJ3,2018,20200.0
+2014,UKG1,2018,2155.0
+2015,UKG1,2018,2290.0
+2016,UKG1,2018,2355.0
+2017,UKG1,2018,2465.0
+2018,UKG1,2018,2260.0
+2014,UKM6,2018,1470.0
+2015,UKM6,2018,1490.0
+2016,UKM6,2018,1605.0
+2017,UKM6,2018,1585.0
+2018,UKM6,2018,1560.0
+2014,UKI4,2018,17885.0
+2015,UKI4,2018,17410.0
+2016,UKI4,2018,17945.0
+2017,UKI4,2018,17920.0
+2018,UKI4,2018,18485.0
+2014,UKI3,2018,29585.0
+2015,UKI3,2018,31365.0
+2016,UKI3,2018,33430.0
+2017,UKI3,2018,35690.0
+2018,UKI3,2018,37610.0
+2014,UKJ4,2018,6805.0
+2015,UKJ4,2018,7190.0
+2016,UKJ4,2018,7395.0
+2017,UKJ4,2018,7470.0
+2018,UKJ4,2018,6990.0
+2014,UKD4,2018,10410.0
+2015,UKD4,2018,11070.0
+2016,UKD4,2018,11490.0
+2017,UKD4,2018,11665.0
+2018,UKD4,2018,11660.0
+2014,UKF2,2018,17595.0
+2015,UKF2,2018,18465.0
+2016,UKF2,2018,19470.0
+2017,UKF2,2018,19685.0
+2018,UKF2,2018,20305.0
+2014,UKF3,2018,2355.0
+2015,UKF3,2018,2800.0
+2016,UKF3,2018,3125.0
+2017,UKF3,2018,3540.0
+2018,UKF3,2018,3880.0
+2014,UKD7,2018,14100.0
+2015,UKD7,2018,15015.0
+2016,UKD7,2018,16135.0
+2017,UKD7,2018,16995.0
+2018,UKD7,2018,17105.0
+2014,UKM5,2018,6310.0
+2015,UKM5,2018,6480.0
+2016,UKM5,2018,6480.0
+2017,UKM5,2018,6465.0
+2018,UKM5,2018,6435.0
+2014,UKE2,2018,6175.0
+2015,UKE2,2018,6280.0
+2016,UKE2,2018,6755.0
+2017,UKE2,2018,7070.0
+2018,UKE2,2018,7395.0
+2014,UKN0,2018,11530.0
+2015,UKN0,2018,11475.0
+2016,UKN0,2018,11430.0
+2017,UKN0,2018,11515.0
+2018,UKN0,2018,11305.0
+2014,UKC2,2018,15730.0
+2015,UKC2,2018,15990.0
+2016,UKC2,2018,16215.0
+2017,UKC2,2018,16605.0
+2018,UKC2,2018,17235.0
+2014,UKI5,2018,4310.0
+2015,UKI5,2018,3915.0
+2016,UKI5,2018,3620.0
+2017,UKI5,2018,3325.0
+2018,UKI5,2018,3350.0
+2014,UKI6,2018,5550.0
+2015,UKI6,2018,5550.0
+2016,UKI6,2018,5005.0
+2017,UKI6,2018,4905.0
+2018,UKI6,2018,4650.0
+2014,UKI7,2018,10005.0
+2015,UKI7,2018,10465.0
+2016,UKI7,2018,10155.0
+2017,UKI7,2018,10870.0
+2018,UKI7,2018,11435.0
+2014,UKG2,2018,7115.0
+2015,UKG2,2018,6910.0
+2016,UKG2,2018,6815.0
+2017,UKG2,2018,7070.0
+2018,UKG2,2018,7125.0
+2014,UKE3,2018,16670.0
+2015,UKE3,2018,17660.0
+2016,UKE3,2018,18445.0
+2017,UKE3,2018,18275.0
+2018,UKE3,2018,18125.0
+2014,UKJ2,2018,16520.0
+2015,UKJ2,2018,16760.0
+2016,UKJ2,2018,17695.0
+2017,UKJ2,2018,18400.0
+2018,UKJ2,2018,19110.0
+2014,UKC1,2018,9610.0
+2015,UKC1,2018,9845.0
+2016,UKC1,2018,10335.0
+2017,UKC1,2018,10570.0
+2018,UKC1,2018,11080.0
+2014,UKM8,2018,21455.0
+2015,UKM8,2018,21915.0
+2016,UKM8,2018,23100.0
+2017,UKM8,2018,23865.0
+2018,UKM8,2018,24695.0
+2014,UKG3,2018,33625.0
+2015,UKG3,2018,35425.0
+2016,UKG3,2018,37125.0
+2017,UKG3,2018,38465.0
+2018,UKG3,2018,39600.0
+2014,UKL1,2018,19460.0
+2015,UKL1,2018,20220.0
+2016,UKL1,2018,21230.0
+2017,UKL1,2018,21335.0
+2018,UKL1,2018,21240.0
+2014,UKE4,2018,22075.0
+2015,UKE4,2018,22455.0
+2016,UKE4,2018,22595.0
+2017,UKE4,2018,22815.0
```

Comment:
In the schema the value is marked as an `int` but values are floats

Reply:
Now integer